src/biogeophys/SectorWaterMod.F90

```diff
@@ -0,0 +1,1238 @@
+          allocate(this%elec_rf_actual_grc (begg:endg))                 ; this%elec_rf_actual_grc             (:)   = 0
+ 
+          allocate(this%input_mon_mfc_withd_grc (begg:endg))            ; this%input_mon_mfc_withd_grc        (:)   = 0
+          allocate(this%input_mon_mfc_cons_grc (begg:endg))             ; this%input_mon_mfc_cons_grc         (:)   = 0
+          allocate(this%mfc_withd_grc (begg:endg))                      ; this%mfc_withd_grc                  (:)   = 0
+          allocate(this%mfc_cons_grc (begg:endg))                       ; this%mfc_cons_grc                   (:)   = 0
+          allocate(this%mfc_withd_actual_grc (begg:endg))               ; this%mfc_withd_actual_grc           (:)   = 0
+          allocate(this%mfc_cons_actual_grc (begg:endg))                ; this%mfc_cons_actual_grc            (:)   = 0
+          allocate(this%mfc_rf_actual_grc (begg:endg))                  ; this%mfc_rf_actual_grc              (:)   = 0
+ 
+          allocate(this%input_mon_min_withd_grc (begg:endg))            ; this%input_mon_min_withd_grc        (:)   = 0
+          allocate(this%input_mon_min_cons_grc (begg:endg))             ; this%input_mon_min_cons_grc         (:)   = 0
+          allocate(this%min_withd_grc (begg:endg))                      ; this%min_withd_grc                  (:)   = 0
+          allocate(this%min_cons_grc (begg:endg))                       ; this%min_cons_grc                   (:)   = 0
+          allocate(this%min_withd_actual_grc (begg:endg))               ; this%min_withd_actual_grc           (:)   = 0
+          allocate(this%min_cons_actual_grc (begg:endg))                ; this%min_cons_actual_grc            (:)   = 0
+          allocate(this%min_rf_actual_grc (begg:endg))                  ; this%min_rf_actual_grc              (:)   = 0
+
+          allocate(this%sectorwater_total_actual_withd (begg:endg))     ; this%sectorwater_total_actual_withd (:)   = 0
+
+     end subroutine SectorWaterInitAllocate
+ 
+ 
+     !-----------------------------------------------------------------------
+     subroutine SectorWaterInitHistory(this, bounds)
+          !
+          ! !DESCRIPTION:
+          ! Initialize sectoral water use history fields
+          !
+          ! !USES:
+          use histFileMod  , only : hist_addfld1d
+          !
+          ! !ARGUMENTS:
+          class(sectorWater_type) , intent(inout) :: this
+          type(bounds_type)      , intent(in)    :: bounds
+          !
+          ! !LOCAL VARIABLES:
+          integer :: begg, endg
+    
+          character(len=*), parameter :: subname = 'InitHistory'
+      !-----------------------------------------------------------------------
+ 
+          begg = bounds%begg; endg= bounds%endg
+
+          ! Add output variables
+          ! Domestic sector:
+          this%input_mon_dom_withd_grc(begg:endg) = 0
+          call hist_addfld1d (fname='INPUT_MON_DOM_WITHD', units='mm', &
```

Comment:
All of the history fields are marked as inactive. Which just means they need to be added to the namelist for them to appear on history. I'm just wondering if when sector water is active -- if *some* of them should be on? Sector water won't be the default at first, so turning on a few makes sense so you don't have to turn them on by hand. At the same time we want to try to pare down the number of active fields, as we already have a ton of output. Any fields that are 3D we need to be especially conservative with, but things that are 2D less so.